credmark/credmark_dev.py

```diff
@@ -433,7 +433,7 @@ def run_model(args):  # pylint: disable=too-many-statements,too-many-branches,to
             else:
                 exit_code = 1
 
-        json_dump(result, sys.stdout)
+        json_dump(result, sys.stdout, indent=4)
```

Comment:
Please a flag for this and do not do it by default. `--format_json`

Reply:
Sure. I have made it an option.